### PR TITLE
fix: Change not listed for sale

### LINKED ISSUE #994: Not available for sell is confusing on the SaleRentActionBox
## 🎉 Description

When there's no open order on a LAND, the text says "Not available for sell" but something like "Currently not listed for sale" is way better.

* [x] 🐛 This is a bug report.
* [ ] 📈 This is a feature request.

### PR DESCRIPTION
**Pull request by LautaroPetaccio**
This PR changes the text `Not available for Sale` for `Currently not listed for sale` as its better than the old one.
Closes #994

### CODE REVIEW

File: webapp/src/modules/translation/locales/en.json [hunk 1]
```diff
@@ -379,7 +379,7 @@
     },
     "sales_rent_action_box": {
       "until": "From {from_date} to {to_date}",
-      "not_for_sale": "Not available for Sale",
+      "not_for_sale": "Currently not listed for sale",
```
Comment (juanmahidalgo) [inline webapp/src/modules/translation/locales/en.json:382]: Should we update the `es.json` file too?

Comment (LautaroPetaccio) [reply webapp/src/modules/translation/locales/en.json:382]: The spanish one seemed correct, as `No disponible para comprar` can be a pretty good translation for `Currently not listed for sale`. We can use something like `Actualmente no disponible para rentar` what do you think?

Comment (juanmahidalgo) [reply webapp/src/modules/translation/locales/en.json:382]: I like the `Actualmente` or maybe appending `en este momento` could be a good option too. I'm fine with both :)

Comment (LautaroPetaccio) [reply webapp/src/modules/translation/locales/en.json:382]: Done! Used `Actualmente no disponible para comprar`

**Review by cyaiox — APPROVED** [review]

**Review by juanmahidalgo — APPROVED** [review]

**Pull request merged**